packages/sdk-react/src/core/queries/useCounterpart.ts

```diff
@@ -441,6 +441,24 @@ export const useDeleteCounterpartContact = () => {
   );
 };
 
+export const useMakeCounterpartContactDefault = () => {
```

Comment:
I disagree with this approach. Consolidating all hooks related to counterparts in a single location enhances code reusability. By doing so, we can avoid redundancy and maintain cleaner, more modular code, rather than embedding all logic within a single component, even if it is only used once

Reply:
I don't see any case where this hook could be needed yet, outside of a component file where it is used.

Reply:
Should we improve our linting to check if a hook is only used once in the same component and, if so, keep it in the same file or component? 

I’m not sure how easy this would be to do, but it seems like a good rule to consider. This might also apply to other things, not just hooks

Reply:
I think the rule is straightforward to follow without a linter.

Write queries in the components by default.
Extract it to a reusable hook if you have to reuse it.